lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -856,21 +856,34 @@ function dosomething_reportback_mbp_request($entity) {
 
   if (module_exists('dosomething_user')) {
     $account = user_load($entity->uid);
+    $node = node_load($entity->nid);
+    $language = dosomething_global_get_language($account, $node);
+
+    $title = $entity->node_title;
+    if ($node) {
+      // Get node title, normal for collections, translatable for campaigns.
+      if (isset($node->title_field)) {
+        $title = $wrapper->language($language)->title_field->value();
+      } else {
+        $title = $node->title;
+      }
+    }
+
     $params = array(
       'email' => $account->mail,
       'uid' => $account->uid,
       'first_name' => dosomething_user_get_field('field_first_name', $account),
-      'campaign_title' => $entity->node_title,
+      'campaign_title' => $title,
       'event_id' => $entity->nid,
       'impact_verb' => $entity->verb,
       'impact_number' => $entity->quantity,
       'impact_noun' => $entity->noun,
       'image_markup' => $image_markup,
     );
-    $node = node_load($entity->nid);
-    if ($node) {
+    if (!empty($node->language)) {
```

Comment:
Should this default to 'en" when there's no value? `campaign_language` should always be set.